dashboard/test/factories/user_level_infos.rb

```diff
@@ -0,0 +1,13 @@
+# == Schema Information
+#
+# Table name: user_level_infos
+#
+#  id            :integer          not null, primary key
+#  time_spent    :integer          default(0)
+#  user_level_id :integer          unsigned
+#
+
+FactoryGirl.define do
+  factory :user_level_info do
+  end
+end
```

Comment:
if you want to define this, I would recommend just putting it in the existing `factories.rb`